src/database/db.py

```diff
@@ -6,7 +6,7 @@
 
 load_dotenv()
 
-DATABASE_URL = os.getenv("DATABASE_URL")
+DATABASE_URL = os.getenv('DATABASE_URL')
```

Comment:
There is no difference between DATABASE_URL = os.getenv("DATABASE_URL") and DATABASE_URL = os.getenv('DATABASE_URL')

Reply:
ok